src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  selected?: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, selected }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
The names of the columns can be written in an array and re-rendered through the `map`.

Reply:
They sure can. Will it be better than it is now? This isn't used anywhere else and is not configurable in any way. Just a static table header.